2.9.0/done/result_mps.md

```diff
@@ -0,0 +1,112 @@
+* bug fixes: All commits that fix bugs and behaviors that do not match the documentation  
+* performance: All commits that are added mainly for performance (we separate this from improvements above to make it easier for users to look for it)  
+* documentation: All commits that add/update documentation  
+* Developers: All commits that are not end-user facing but still impact people that compile from source, develop into pytorch, extend pytorch, etc  
+* not user facing: All commits that are not public end-user facing and hence should be dropped from the release notes
+
+## mps
+
+### bc breaking
+
+- Build metal kernels of MacOS-14+ ([\#159733](https://github.com/pytorch/pytorch/pull/159733))  
+- Remove all pre-MacOS14 logic ([\#159912](https://github.com/pytorch/pytorch/pull/159912))
+
+### deprecation
+
+### new features
+
+### improvements
+
+- Add `shifted_chebyshev_polynomial_[tuvw], simd_[arg][max|min]`,  `igamma/igammac,grid_sampler_3d, native_dropout`/`native_dropout_backward`  ([\#157488](https://github.com/pytorch/pytorch/pull/157488), [\#158990](https://github.com/pytorch/pytorch/pull/158990), [\#161927](https://github.com/pytorch/pytorch/pull/161927), [\#160541](https://github.com/pytorch/pytorch/pull/160541), [\#162108](https://github.com/pytorch/pytorch/pull/162108))  
+- For sparse tensors: Add coalesce, `indices/values,sgn/asinh/atanh/asin/atan/ceil/erf/expm1/floor/frac/isnan/nan_to_num/log1p/rad2deg/deg2rad/neg/round/relu/sin/sinh/sqrt/tan/tanh/sign/signbit/isinf/isposinf/isneginf,cat` ([\#159729](https://github.com/pytorch/pytorch/pull/159729)/[\#160254](https://github.com/pytorch/pytorch/pull/160254), [\#160223](https://github.com/pytorch/pytorch/pull/160223),[\#161846](https://github.com/pytorch/pytorch/pull/161846),[\#162007](https://github.com/pytorch/pytorch/pull/162007))  
+- Extend atomic operations to all int types ([\#158179](https://github.com/pytorch/pytorch/pull/158179))  
+- Extend `index_put` to complex types ([\#160159](https://github.com/pytorch/pytorch/pull/160159))  
+- Extend addmm to integral types ([\#160270](https://github.com/pytorch/pytorch/pull/160270))  
+- Add support for unsigned types ([\#159094](https://github.com/pytorch/pytorch/pull/159094))  
+- Add API to query GPU core count ([\#160414](https://github.com/pytorch/pytorch/pull/160414))  
+- Update `avg_pool3d` kernel to use `opmath_t` ([\#161071](https://github.com/pytorch/pytorch/pull/161071))  
+- Add slow version of `kthvalue` ([\#161817](https://github.com/pytorch/pytorch/pull/161817))  
+- Type-promote tensor-iterator common dtype ([\#160334](https://github.com/pytorch/pytorch/pull/160334))  
+- Add fused\_rms and sdpa\_mps fallback ops for AOTInductor ([\#156844](https://github.com/pytorch/pytorch/pull/156844))  
+- Implement logcumsumexp metal kernel ([\#156858](https://github.com/pytorch/pytorch/pull/156858))  
+- Migrate round unary op to Metal ([\#161712](https://github.com/pytorch/pytorch/pull/161712))  
```

Comment:
That should go into bugfixes, because it fixes the behavior for `X.5`/`-X.5` cases